### ISSUE TITLE
overview.md -- clarify how to open anaconda and launch jupyter notebook

### ISSUE THREAD

**Issue by kchatlosh** [DHRI-Curriculum/text-analysis#10]
Referring to [overview.md](https://github.com/DHRI-Curriculum/text-analysis/blob/master/sections/overview.md), under sub-heading [Setup and installation](https://github.com/DHRI-Curriculum/text-analysis/blob/master/sections/overview.md#setup-and-installation):

Unpack this a bit: "Find Anaconda on your computer, Launch a Jupyter Notebook." -- should they open the command line and then enter a specific command to open anaconda? and then enter another command to launch a jupyter notebook?

Sorry I missed this while you (@rachelrakov) got started and said it out loud / showed it, and then couldn't figure it out on my own, so thought it'd be a good thing to add in here :)

**Comment by kallewesterling** [comment]
(added in some comments/details/links in your question but letting @rachelrakov take the lead on this)

**Comment by kallewesterling** [comment]
Added some clarification [here](https://github.com/DHRI-Curriculum/text-analysis/commit/fda5a93c0035ce062269341de4cd1527a51b2b28) and [here](https://github.com/DHRI-Curriculum/text-analysis/commit/06ae4dfa22a92f669f4c40f288060dc6baeb2a30).

I believe the "Anaconda on your computer" refers to the GUI interface of Anaconda, which is found as "Anaconda Navigator" in the Applications folder on the learner's computer. I added something [here](https://github.com/DHRI-Curriculum/text-analysis/commit/22730f3441cd2885b8d6a3ac4c2184080196d227) about it.

@rachelrakov - as with all my edits, feel free to reverse or edit as you see fit.
